locales/sv_SE/pcsx2_Main.po

```diff
@@ -899,32 +899,30 @@ msgstr "Om %s"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:54
 msgid "Website"
-msgstr ""
+msgstr "Webställe"
```

Comment:
```suggestion
msgstr "Webbplats"
```

Reply:
Makes no difference to me. - I'll change it.